ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_nat.schema.yml

```diff
@@ -0,0 +1,180 @@
+      synchronization:
+        type: dict
+        keys:
+          description:
+            type: str
+          expiry_interval:
+            type: int
+            convert_types:
+              - str
+            min: 60
+            max: 3600
+            description: in seconds
+          local_interface:
+            type: str
+            description: EOS interface name
+          peer_address:
+            type: str
+            description: IPv4 address
+          port_range:
+            type: dict
+            keys:
+              first_port:
+                type: int
+                convert_types:
+                  - str
+                min: 1024
+                max: 65535
+              last_port:
+                type: int
+                convert_types:
+                  - str
+                min: 1024
+                max: 65535
+                description: '>= first_port'
+              split_disabled:
+                type: bool
+          shutdown:
+            type: bool
+      translation:
+        type: dict
+        keys:
+          address_selection:
+            type: dict
+            keys:
+              any:
+                type: bool
+              hash_field_source_ip:
+                type: bool
+          counters:
+            type: bool
+          low_mark:
+            type: dict
+            keys:
+              percentage:
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 99
+              host_percentage:
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 99
+          max_entries:
+            type: dict
+            keys:
+              limit:
+                type: int
+                convert_types:
+                  - str
+                min: 0
+                max: 4294967295
+              host_limit:
+                type: int
+                convert_types:
+                  - str
+                min: 0
+                max: 4294967295
+              ip_limits:
+                type: list
+                primary_key: ip
+                convert_types:
+                  - dict
+                items:
+                  type: dict
+                  keys:
+                    ip:
+                      type: str
+                      description: IPv4 address
+                    limit:
+                      type: int
+                      required: true
+                      convert_types:
+                        - str
+                      min: 0
+                      max: 4294967295
+          timeouts:
+            type: list
+            primary_key: protocol
+            convert_types:
+              - dict
+            items:
+              type: dict
+              keys:
+                protocol:
+                  type: str
+                  valid_values: ["tcp", "udp"]
+                timeout:
```

Comment:
Please add a description like `Timeout in seconds` or whatever unit is applicable here.